content/themes/ediblesage/page-new_plot_form.php

```diff
@@ -1,9 +1,8 @@
+<div class="modal fade" id="basicModal" tabindex="-1" role="dialog" aria-labelledby="basicModal" aria-hidden="true">
 <!-- ok so general rule is don't touch anything with ng- prefix -->
 <!-- whatever else is cool -->
 <!-- sorry about the messy code for doing the columns -->
 <!-- you can probably do this with sass much easier?? -->
-
-<div class="modal fade" id="basicModal" tabindex="-1" role="dialog" aria-labelledby="basicModal" aria-hidden="true">
     <div class="modal-dialog">
         <div class="modal-content">
 
```

Comment:
this file change is no really to do with the map pop up formatting stuff but was a fix in the plot form template that got committed in this branch